Package.swift

```diff
@@ -25,10 +25,10 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-log.git", from: "1.4.0"),
         .package(url: "https://github.com/apple/swift-metrics.git", "1.0.0"..<"3.0.0"),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.16.1"),
-        .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.7.2"),
-        .package(url: "https://github.com/apple/swift-nio-transport-services.git", from: "1.0.0"),
-        .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.3.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", .upToNextMajor(from: "2.32.0")),
```

Comment:
Any particular reason to this being `upToNextMajor`? I seem to remember it moving to `from` in a previous PR. Either way the whole manifest should be consistent

Reply:
They were originally all `upToNextMajor`. I converted to `from` as it seemed more common. There is a possibility I did this after the async-await branch was created and then each time I would rebase with main it would jump about. Will revert to `from`